projects/explorer/src/app/pages/blocks/blocks.component.ts

```diff
@@ -20,27 +20,40 @@ export class BlocksComponent implements OnInit {
   defaultPageSize = this.pageSizeOptions[1];
   defaultPageNumber = 1;
 
-  pollingInterval = 30;
+  pollingInterval = 5;
   latestBlock$: Observable<InlineResponse20035 | undefined>;
   latestBlockHeight$: Observable<bigint | undefined>;
   firstBlockHeight$: Observable<bigint | undefined>;
   blocks$: Observable<InlineResponse20036[] | undefined>;
 
+  autoEnabled: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+  blocksEmpty: boolean;
+
   constructor(
     private router: Router,
     private route: ActivatedRoute,
     private cosmosSDK: CosmosSDKService,
   ) {
-    const timer$ = timer(0, this.pollingInterval * 1000);
-    const sdk$ = timer$.pipe(mergeMap((_) => this.cosmosSDK.sdk$));
-    this.latestBlock$ = sdk$.pipe(
-      mergeMap((sdk) => rest.tendermint.getLatestBlock(sdk.rest).then((res) => res.data)),
+    this.blocksEmpty = true;
```

Comment:
flag for detect that user come to this page now

Reply:
I think `isFirstAccess` is better than `blocksEmpty`.